pkg/resource/v1/certconfig/create.go

```diff
@@ -0,0 +1,9 @@
+package certconfig
+
+import (
+	"context"
+)
+
+func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
```

Comment:
I will add godoc comments to exported functions when implementing corresponding functions. I feel that this way I can produce more useful content there.